res/mixxx.qrc

```diff
@@ -3,6 +3,7 @@
         <file>../LICENSE</file>
         <file>images/ic_mixxx_window.png</file>
         <file>images/library/cover_default.png</file>
```

Comment:
remove the PNG image here and delete the PNG file

Reply:
would require all other instances to be replaced with the svg file. Suppose its doable.